sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -774,19 +868,28 @@ static int create_sdw_codec_dai(struct device *dev,
 		if (!codec_info_list[idx].direction[i])
 			continue;
 
+		/* create stream name based on the first link id */
 		name = devm_kasprintf(dev, GFP_KERNEL,
-				      sdw_stream_name[i], link_id);
+				      sdw_stream_name[i], cpu_dai_id[0]);
```

Comment:
Can we include all cpu ids in the stream name? For example SDW0,1-Playback

Reply:
@bardliao , do you have a PR for topology ?  Machine driver need to work with topology file.

Reply:
> @bardliao , do you have a PR for topology ? Machine driver need to work with topology file.

No, we don't have a topology for multi cpu now. So, we can set any stream name if we want.